src/standalone/InlineSwitch/index.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback } from "react";
+import { Grid, Typography, withStyles, Theme, Switch } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+export interface InlineSwitch {
+	/**
+	 * Value for switch position
+	 */
+	switchValue: boolean;
+	/**
+	 * Set value for switch position
+	 */
+	setSwitchValue: (checked: boolean) => void;
+	/**
+	 * Display switch control?
+	 */
+	displaySwitch: boolean;
+	/**
+	 * Label for switch control (only used if displaySwitch is truthy)
+	 */
+	switchLabel?: React.ReactNode;
+	children?: React.ReactElement;
+}
+
+const useStyles = makeStyles(
+	{
+		switch: {
+			lineHeight: "30px",
+			float: "right",
+			marginTop: -30,
+		},
+		labelWithSwitch: {
+			marginTop: 15,
+		},
+	},
+	{ name: "InlineSwitch" }
+);
+
+const AntSwitch = withStyles((theme: Theme) => ({
+	root: {
+		width: 35,
+		height: 16,
+		padding: 0,
+		display: "flex",
+	},
+	switchBase: {
+		padding: 2,
+		color: theme.palette.grey[500],
+		"&$checked": {
+			transform: "translateX(18px)",
+			color: theme.palette.common.white,
+			"& + $track": {
+				opacity: 1,
+				backgroundColor: theme.palette.primary.main,
+				borderColor: theme.palette.primary.main,
+			},
+		},
+	},
+	thumb: {
+		width: 12,
+		height: 12,
+		boxShadow: "none",
+	},
+	track: {
+		border: `1px solid ${theme.palette.grey[500]}`,
+		borderRadius: 16 / 2,
+		opacity: 1,
+		backgroundColor: theme.palette.common.white,
+	},
+	checked: {},
+}))(Switch);
+
+const InlineSwitch = (props: InlineSwitch) => {
```

Comment:
you should refactor MultiSelectWithTags component to use this instead

Reply:
- @TQjona , Updated changes. Please Review.